packages/tokamak/contracts-bedrock/scripts/verify-predeploy-contracts.sh

```diff
@@ -125,7 +124,6 @@ contracts+=( ["4200000000000000000000000000000000000001"]=""  # Proxy
              ["420000000000000000000000000000000000001a"]=""  # L1FeeVault
              ["4200000000000000000000000000000000000020"]=""  # SchemaRegistry
              ["4200000000000000000000000000000000000021"]=""  # EAS
-             ["4200000000000000000000000000000000000486"]=""  # ETH (TOKAMAK)
```

Comment:
I think you don't need to have any changes in this line. Could you restore it? ETH is predeploy contract

Reply:
yes! i reverted manually